packages/taquito/src/taquito.ts

```diff
@@ -336,6 +343,23 @@ export class TezosToolkit {
     }
   }
 
+  /**
+   * @description Sets injector provider on the Tezos Taquito instance
+   *
+   * @param options Injector to use to interact with the Tezos network by default RpcInjector
+   *
+   */
+  setInjectorProvider(injectorProvider?: SetProviderOptions['injectorProvider']) {
+    if (!this._options.injectorProvider && typeof injectorProvider === 'undefined') {
+      const p = new RpcInjector(this._context);
```

Comment:
small nitpick, but looking at the other set_Provider code, wouldn't the variable `p` be `i` instead since 'injector' starts with an `i`

Reply:
good point, will update